src/stylesheets/components/_forms.scss

```diff
@@ -1,3 +1,5 @@
+// scss-lint:disable QualifyingElement
```

Comment:
Another alternative is to remove `button` and `input` from the selectors (may also help with specificity issues). @msecret what do you think?